modules/quadbin/snowflake/doc/QUADBIN_TOZXY.md

```diff
@@ -0,0 +1,25 @@
+### QUADBIN_TOZXY
+
+{{% bannerNote type="code" %}}
+carto.QUADBIN_TOZXY(quadbin)
+{{%/ bannerNote %}}
+
+**Description**
+
+Returns the zoom level `z` and coordinates `x`, `y` for a given quadbin.
+
+* `quadbin`: `INT` quadbin we want to extract tile information from.
```

Comment:
For `quadint` we were using the type BIGINT for everything quadint related. This is the same case. But it seems that they are synonyms so we can keep it if you we are not losing precision